### DISCUSSION TITLE
How to exclude a Babel plugin? / How to use BigInt exponentiation?

### DISCUSSION THREAD

**Discussion by hazae41** [vercel/next.js#24611] [Help] [answered]
Similarly to [this issue on Gatsby](https://github.com/gatsbyjs/gatsby/issues/25297), I can't exclude the same plugin from Babel.

I DON'T want to transform `10n ** 18n` to `Math.pow(10n, 18n)`

I tried the following on .babelrc
```
{
  "presets": ["next/babel", {
    "exclude": ["babel-plugin-transform-exponentiation-operator"]
  }],
  "plugins": []
}
```

I also tried upgrading to Next 10.2.0 and cleaning the .next folder

Here are the logs
```
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
info  - Using webpack 5. Reason: future.webpack5 option enabled https://nextjs.org/docs/messages/webpack5
info  - Using external babel configuration from <my directory>\.babelrc
```

However, I still get the following error:
![Capture](https://user-images.githubusercontent.com/4405263/116600465-a01de880-a929-11eb-805b-f2e1ede96d6d.PNG)

Any idea? Thanks

**Comment by bobarros** [comment, marked as answer]
First thing, is because you need to exclude from preset-env. 
The second, you can't write the full name. You need to exclude "babel-plugin-"

You have to do it like so:

```json
"presets": [
        [
          "next/babel",
          {
            "preset-env": {
              "exclude": ["transform-exponentiation-operator"]
            }
          }
        ]
      ]
```

All the best.

**Reply by hazae41** [reply]
It works, thanks a lot!

**Reply by bobarros** [reply]
No problem. 

You invested some time looking for the answer and it seems that is not common out there.
Might be a good idea to mark this one as the answer. 
It will increase the chance to help others to find it.
What do you think?

**Reply by humble-barnacle001** [reply]
Hey, how to do the same in SWC since it seems to be the default build tool for next.js now?
